crates/intrinsic-test/src/json_parser.rs

```diff
@@ -17,6 +17,7 @@ struct ReturnType {
 #[serde(untagged, deny_unknown_fields)]
 pub enum ArgPrep {
     Register {
+        #[allow(dead_code)]
         #[serde(rename = "register")]
         #[allow(dead_code)]
```

Comment:
@Amanieu just noticed a bit of a merge/rebase failure here, seems this attribute was duplicated